docs/visual-basic/programming-guide/concepts/async/walkthrough-accessing-the-web-by-using-async-and-await.md

```diff
@@ -15,35 +15,35 @@ If you don't want to build the applications yourself, you can download "Async Sa
 
 In this walkthrough, you complete the following tasks:
 
-- [To create a WPF application](#CreateWPFApp)
+- [Create a WPF application](#CreateWPFApp)
```

Comment:
Perhaps add this to a checklist like in the mvc style?
https://github.com/dotnet/docs/blame/master/docs/machine-learning/tutorials/movie-recommendation.md#L17
https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/movie-recommendation

Then we could also get rid of the custom anchors.

Reply:
Thanks, @Maira. The checklist looks great, and I've gotten rid of all the custom anchors.